# HG changeset patch
# Parent b5e94cf0b3662953666cd7fa691fbc13db9fa1b7
# Any change in <KERNEL_PATH>/include/uapi should be dynamic into <KERNEL_PATH>/user_headers

diff --git a/include/kernel-defaults.mk b/include/kernel-defaults.mk
--- a/include/kernel-defaults.mk
+++ b/include/kernel-defaults.mk
@@ -119,7 +119,7 @@ define Kernel/Configure/Default
 	)
 	-$(_SINGLE)yes ""|$(MAKE) $(KERNEL_MAKEOPTS) oldconfig prepare scripts 	
 	rm -rf $(KERNEL_BUILD_DIR)/modules
-	$(_SINGLE) [ -d $(LINUX_DIR)/user_headers ] || $(MAKE) $(KERNEL_MAKEOPTS) INSTALL_HDR_PATH=$(LINUX_DIR)/user_headers headers_install
+	$(_SINGLE) $(MAKE) $(KERNEL_MAKEOPTS) INSTALL_HDR_PATH=$(LINUX_DIR)/user_headers headers_install
 	$(SH_FUNC) grep '=[ym]' $(LINUX_DIR)/.config | LC_ALL=C sort | md5s > $(LINUX_DIR)/.vermagic
 endef
 
